docs/src/index.md

```diff
@@ -11,7 +11,7 @@ The general form of the optimization problem is
 ```math
 \begin{aligned}
 \min \quad & f(x) \\
-& c_i(x) = 0, \quad i \in E, \\
+& c_i(x) = c_{V_i}, \quad i \in E, \\
```

Comment:
I'm unsure this is the best solution, feel free to suggest alternatives. I considered jumping straight to the general statement, but it is so common to separate out equalities and inequalities that I do see value in the way things are stated now.

Reply:
I like your suggestion. Does the `V` stand for something? Otherwise, I would suggest `c_{E_i}` for Equality, which would match `i \in E`.